srcpkgs/librepcb/template

```diff
@@ -1,17 +1,23 @@
 # Template file for 'librepcb'
 pkgname=librepcb
-version=1.0.0
+version=1.3.0
 revision=1
 build_style=cmake
 configure_args="-DBUILD_QTQUICK_TEST=OFF"
-hostmakedepends="unzip qt5-qmake qt5-host-tools"
-makedepends="zlib-devel qt5-devel qt5-tools-devel qt5-plugin-mysql
- qt5-plugin-odbc qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-tds
- qt5-svg-devel occt-devel glu-devel openssl-devel"
-depends="qt5-plugin-sqlite"
+hostmakedepends="unzip qt6-base qt6-tools rust cargo"
+makedepends="zlib-devel qt6-base-devel qt6-tools-devel qt6-plugin-mysql
+ qt6-plugin-odbc qt6-plugin-pgsql qt6-plugin-sqlite
+ qt6-svg-devel occt-devel glu-devel openssl-devel rust cargo"
```

Comment:
should cargo be in makedepends or hostmakedepends?

Reply:
iirc it wasn't working with it in just 1 for arm. only host was bombing out about not finding the arch and to run some rust build for it. and i think it doesnt pass config or first build steps with it in makedeps

Reply:
it's probably just `rust-std` that's needed on target. and cargo depends on rust so having both is redundant